robot-installer.sh

```diff
@@ -19,7 +44,6 @@ function latest_cirrus(){
     echo "$VERSION"
 }
```

Comment:
It should really come from pypi I suppose, but not sure how that jives with a custom pypi, so for now leaving GH as the source of truth, but another TBD issue